src/Directory.Packages.props

```diff
@@ -23,7 +23,8 @@
 		<PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
 		<PackageVersion Include="Microsoft.UI.Xaml" Version="2.7.1" />
 		<PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-	    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.4.231219000" />
+		<PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.4.231219000" />
```

Comment:
@dansiegel based on the current CI errors I imagine you will need to bump the Microsoft.WindowsAppSDK version.
```
Error NU1605: Warning As Error: Detected package downgrade: Microsoft.WindowsAppSDK from 1.5.240227000 to 1.4.231219000. Reference the package directly from the project to select a different version. 
 Uno.Extensions.Maui.WinUI -> Microsoft.Graphics.Win2D 1.2.0 -> Microsoft.WindowsAppSDK (>= 1.5.240227000) 
 Uno.Extensions.Maui.WinUI -> Microsoft.WindowsAppSDK (>= 1.4.231219000)
```